std/net/curl.d

```diff
@@ -2261,32 +2261,46 @@ struct HTTP
     {
         p.status.reset();
 
+        CurlOption opt;
+        scope (exit)
```

Comment:
Yes. p.curl.perform may throw. but I think that this scope(exit) statement is not so good too...